CHANGELOG/CHANGELOG-0.12.md

```diff
@@ -0,0 +1,206 @@
+# Release notes for v0.12.0
+
+# Changelog since v0.11.0
+
+## Changes by Kind
+
+### Other (Cleanup or Flake)
+
+- Leader election is only supported for leases via NewLeaderElection. ([#109](https://github.com/kubernetes-csi/csi-lib-utils/pull/109), [@pohly](https://github.com/pohly))
```

Comment:
It looks like this change is already part of v0.11.0. Not sure why it got picked up by the tool. https://github.com/kubernetes-csi/csi-lib-utils/blob/master/CHANGELOG/CHANGELOG-0.11.md

Reply:
Updated!

Reply:
The start sha should be f89955e5695f1d0d2dcaa45939c0e57e9ea6cf59.  The start sha used in this run was before the 0.11 release.

Reply:
I changed to use tags, this command works better:

```
release-notes \
  --org=kubernetes-csi \
  --repo=csi-lib-utils \
  --start-rev=release-0.11 \
  --discover=mergebase-to-latest \
  --required-author="" \
  --markdown-links \
  --output out.md
```

Reply:
I think `--start-rev` is ignored if you use `--discover`.